src/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+import "./Nav.scss";
+import Home from "../../assets/icons/Home.svg";
```

Comment:
These are not components so should be lowercase hyphenated as they are files.
- BEM for styles
- PascalCasing for components
- camelCasing for ts
- lowercase-hypenated for other files e.g. images assets. 

🤯 

You will need to update the file names aswell as the imports e.g. `home.svg`